kerosene/nn/init.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Kerosene Authors. All Rights Reserved.
+#
+# Licensed under the MIT License;
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://opensource.org/licenses/MIT
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from enum import Enum
+from typing import Union, Callable
+
+from torch.nn import init
+
+
+class InitializerType(Enum):
```

Comment:
I think this is out of the scope of the library as its model related.

Reply:
Models are in SAMITorch. What if the user want different initialization ? Should be initialize them in SAMITorch or let the user init the nets with Kerosene ?